templates.toml

```diff
@@ -49,7 +49,7 @@ template = "axum"
 [starters.bevy-hello-world]
 title = "Bevy"
 description = "Data driven game engine that compiles to WASM"
-path = "bevy/hello-world"
+path = "bevy/hello-world/"
```

Comment:
why?

Reply:
oops, might be an autoformatting artifact :sweat_smile: